Tasks/QuickPerfTest/task.json

```diff
@@ -110,6 +119,22 @@
                 "2": "Self-provisioned agents"
             }
         },
+        {
+            "name": "selfProvisionedRig",
```

Comment:
>name [](start = 13, length = 4)

@PBoraMSFT  Do we want to show the available agent groups (through a connected api)

Reply:
It would be nice to offer a combo box like one see in the Edit load test VSTS page. It is outside the scope of my PR.

Reply:
@cltshivash - yes, a combo-box with available agent group names would be nice, but we should make it an editable combo-box to allow users to also type in another name (e.g., a pipeline variable that gets an agent group name because the rig was setup in the pipeline itself in a previous step).